src/openforms/js/components/admin/form_design/variables/VariablesTable.js

```diff
@@ -104,84 +106,108 @@ const EditableVariableRow = ({index, variable, onDelete, onChange}) => {
     setPrefillAttributeChoices(response.data.map(attribute => [attribute.id, attribute.label]));
   }, [variable.prefillPlugin]);
 
+  const hasErrors = Object.entries(variable.errors || {}).length;
+
   return (
-    <tr className={`row${(index % 2) + 1}`}>
+    <tr
+      className={classNames('variables-table__row', `row${(index % 2) + 1}`, {
+        'variables-table__row--errors': hasErrors,
+      })}
+    >
       <td>
         <DeleteIcon onConfirm={() => onDelete(variable.key)} message={deleteConfirmMessage} />
       </td>
       <td>
-        <TextInput
-          name="name"
-          value={variable.name}
-          onChange={onValueChanged}
-          onBlur={updateKey}
-          noVTextField={true}
-        />
+        <Field name="name" errors={variable.errors?.name}>
+          <TextInput
+            name="name"
+            value={variable.name}
+            onChange={onValueChanged}
+            onBlur={updateKey}
+            noVTextField={true}
```

Comment:
```suggestion
            noVTextField
```
this is equivalent for boolean props